drivers/nrf24l01p/include/diagnostics.h

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
So far, in `nrf24ctl` it is available, if the module `nrf24l01p_diagnostics` is used.
I wrote that for development to check if I see the register values I expect.
For development it was quiet useful for me and I think for others it will bee too.

If I understand you right, you suggest that I don´t make it a pseudomodule and the developer who wants to check register values adds the function call on the fly? And if it is not used by the developer or the application it will be garbage collected. If so, I aggree.
But the function definitions would still be a part of the driver, right?